doc/previewflag.md

```diff
@@ -0,0 +1,18 @@
+# ASAB WebUI PreviewFlag component
+
+`PreviewFlag` is a react reusable component.
+It is used to display a customizable (preview) flag. 
+
+# Setup
+
+To use the preview flag, pass the `name` prop to the component.
```

Comment:
@queuing4oranges It is also good to mention, that the `name` should be of `string` type